models/base/scratch/databricks/snowplow_media_player_base_events_this_run.sql

```diff
@@ -11,18 +11,23 @@
 -- check for exceptions
 {% if var("snowplow__enable_whatwg_media") is false and var("snowplow__enable_whatwg_video") %}
   {{ exceptions.raise_compiler_error("variable: snowplow__enable_whatwg_video is enabled but variable: snowplow__enable_whatwg_media is not, both need to be enabled for modelling html5 video tracking data.") }}
-{% elif not var("snowplow__enable_youtube") and not var("snowplow__enable_whatwg_media") %}
-  {{ exceptions.raise_compiler_error("No media context enabled. Please enable as many of the following variables as required: snowplow__enable_youtube, snowplow__enable_whatwg_media, snowplow__enable_whatwg_video") }}
+{% elif not var("snowplow__enable_youtube") and not var("snowplow__enable_whatwg_media") and not var("snowplow__enable_media_player_v2") %}
```

Comment:
I tried running it with neither `snowplow__enable_media_player_v2` or `snowplow__enable_media_player_v1` enabled, although it makes sense to check one, we could make it more fool proof if we added another quick check based on that as there are many config options to confuse people.

Reply:
Good point! Have added a check for at least one media player context to be enabled.